src/main/scala/io/epiphanous/flinkrunner/model/sink/JdbcSinkConfig.scala

```diff
@@ -10,17 +10,19 @@ import io.epiphanous.flinkrunner.operator.CreateTableJdbcSinkFunction
 import io.epiphanous.flinkrunner.util.SqlBuilder
 import org.apache.avro.generic.GenericRecord
 import org.apache.flink.api.common.functions.RuntimeContext
-import org.apache.flink.api.common.typeinfo.TypeInformation
+import org.apache.flink.api.common.typeinfo.{TypeHint, TypeInformation}
```

Comment:
I usually run Optimize Imports in IntelliJ over all the source to make sure uneeded imports are cleaned up, followed by a "scalafmt" in the cli to make sure code is consistently formatted.

Reply:
we can create a task that does the latter - is there a command for the "optimize imports"?

Reply:
@LizaMo there's a [command](https://stackoverflow.com/questions/8608710/intellij-organize-imports/8608740#8608740) this article might be helpful. In the GUI is under optimize imports as well.